public/static/js/patch_new.js

```diff
@@ -94,6 +94,18 @@ mciModule.controller('PatchController', function($scope, $filter, $window, notif
     }
   }
 
+  $scope.deleteFromQueue = function() {
+    var successHandler =  function(resp) {
+        window.location.reload();
+        notificationService.pushNotification("Item successfully cleared from queue", 'success');
```

Comment:
This banner doesn't show up. Is there something about reloading that does weird things?

Reply:
My guess would be that reloading stops execution where it is so this line never gets executed.

Reply:
pushFlash is a go function for the ui server though and this is a v2 route, so can this not be done? It's not strictly necessary, I just thought it'd be nice.